src/PostgREST/Config.hs

```diff
@@ -42,7 +42,7 @@ import qualified Data.Map.Strict        as M
 
```

Comment:
can't comment on the hidden line above (the `Map` import). If that is still used after the split-up into `CLI`, the same reasoning for naming etc. applies as I commented there.